zap/src/test/java/org/zaproxy/zap/extension/ascan/filters/impl/GenericFilterUtilityTest.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.ascan.filters.impl;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.zaproxy.zap.WithConfigsTest;
+import org.zaproxy.zap.extension.ascan.filters.FilterCriteria;
+import org.zaproxy.zap.extension.ascan.filters.FilterResult;
+import org.zaproxy.zap.model.StructuralNode;
+
+/** @author KSASAN preetkaran20@gmail.com */
+public class GenericFilterUtilityTest extends WithConfigsTest {
+
+    private AbstractGenericScanFilter<String> abstractGenericScanFilter;
+
+    @Before
+    public void init() {
+        abstractGenericScanFilter =
+                new AbstractGenericScanFilter<String>() {
+
+                    @Override
+                    public FilterResult isFiltered(StructuralNode node) {
+                        return null;
+                    }
+
+                    @Override
+                    public String getFilterType() {
+                        return null;
+                    }
+                };
+    }
+
+    @Test
+    public void testEmptyMessageValuesAndFilterValuesIncludeCriteria() {
```

Comment:
Was not suggesting replacing by "messages" but completely remove it, here we don't care of what the values might be.

Reply:
sure.

Reply:
Was referring to the method name not the whole method (same for the other methods and variables which are using messages).

Reply:
didn't got your point. can you please explain.

Reply:
are you asking to remove the "message" word from the method ?

Reply:
not sure if that has any impact anywhere, plus removing message might not fit for all the methods. IMHO i think this message keyword will not cause any harm.

Reply:
Yes, it was to remove the "message" from the method name (and variables). That change is for us, these tests don't have anything related to (HTTP) messages so shouldn't need to say that ("values" is enough IMO).

Reply:
ok. i have done the change.